packages/api/Tests/UnitTests/Flexihours/GetOvertimeTests.cs

```diff
@@ -0,0 +1,218 @@
+﻿using AlvTime.Persistence.DataBaseModels;
+using AlvTime.Persistence.Repositories;
+using System;
+using System.Linq;
+using AlvTime.Business.FlexiHours;
+using Xunit;
+
+namespace Tests.UnitTests.Flexihours
+{
+    public class GetOvertimeTests
+    {
+        private readonly AlvTime_dbContext _context = new AlvTimeDbContextBuilder()
+                .WithUsers()
+                .CreateDbContext();
+
+        private readonly DateTime _startDate = new DateTime(2020, 01, 02);
+
+        [Fact]
+        public void GetFlexhours_NoWorkAtAll_AvailableIs0Overtime0Flex()
+        {
+            FlexhourStorage calculator = CreateStorage();
+            var flexhours = calculator.GetAvailableHours(1);
+
+            Assert.Equal(0, flexhours.TotalHours);
```

Comment:
Maybe we could rename the properties to "AvailableHoursBeforeCompensation" and "AvailableHoursAfterCompensation" instead of "TotalHours" and "TotalHoursIncludingCompensation"?